src/Bicep.Core.RegistryClient/DownloadManifestResult.cs

```diff
@@ -8,11 +8,14 @@ namespace Bicep.Core.RegistryClient
 {
     public class DownloadManifestResult
     {
-        internal DownloadManifestResult(Stream content)
+        public DownloadManifestResult(string digest, Stream content)
         {
+            Digest = digest;
             Content = content;
         }
 
+        public string Digest { get; }
```

Comment:
>Digest [](http://example.com/codeflow?start=22&length=6)

@annelo-msft This is the additional change I made. In the case where I download a manifest by tag, this allows me to validate the digest of received content.